src/ucp/wireup/wireup_cm.c

```diff
@@ -534,7 +445,6 @@ static unsigned ucp_ep_cm_disconnect_progress(void *arg)
         ucs_assert(!(ucp_ep->flags & UCP_EP_FLAG_CLOSE_REQ_VALID));
     } else if (ucp_ep->flags & UCP_EP_FLAG_LOCAL_CONNECTED) {
         /* if the EP is local connected, need to flush it from main thread first */
-        ucp_ep_cm_remote_disconnect_progress(ucp_ep);
```

Comment:
do we need to call ucp_ep_cm_disconnect_cm_lane() as if flush is completed?
@evgeny-leksikov wdyt?

Reply:
I don't think so, because the only thing which is allowed and mandatory after err callback - close the EP (and only forced starting from this PR)

Reply:
remove invoking err_cb?

Reply:
so we rely on use to close the ep (from err callback) which will call ucp_ep_cm_disconnect_cm_lane?

Reply:
yes, actually it does not matter disconnect or close CM lane